tiledb/sm/filesystem/vfs.h

```diff
@@ -581,6 +581,28 @@ class VFS : private VFSBase, protected S3_within_VFS {
     return results;
   }
 
+  /**
+   * Lists objects and object information that start with `prefix`, invoking
```

Comment:
Nit to update comment, even though the name for the function documents itself.

```suggestion
   * Recursively lists objects and object information that start with `prefix`, invoking
```